src/tests/ftest/telemetry/pool_svc_metrics.py

```diff
@@ -0,0 +1,133 @@
+'''
+  (C) Copyright 2024 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+'''
+import time
+
+from telemetry_test_base import TestWithTelemetry
+
+MAP_VERSION_METRIC = "engine_pool_svc_map_version"
+SVC_LEADER_METRIC = "engine_pool_svc_leader"
+DEGRADED_RANKS_METRIC = "engine_pool_svc_degraded_ranks"
+
+
+class PoolServiceMetrics(TestWithTelemetry):
+    """Verify pool service metric values.
+
+    :avocado: recursive
+    """
+
+    def collect_svc_telemetry(self, pool_uuid):
+        """Collect the pool service metric values.
+
+        Args:
+            pool_uuid: The UUID of the pool to collect metrics from.
+
+        Returns:
+            dict: A dict of the current pool service leader's metrics.
+        """
+        def _map_version(rank_metrics):
+            if rank_metrics is None or MAP_VERSION_METRIC not in rank_metrics:
+                return 0
+            return rank_metrics[MAP_VERSION_METRIC]
+
+        def _pool_rank_metrics(host_metrics, pool_uuid):
+            self.log.debug("collecting rank metrics for pool: %s", pool_uuid)
+            pm = {}
+            for hm in host_metrics.values():
+                for k, v in hm.items():
```

Comment:
We used to have a lint check for this and I'm not sure why it's not flagged anymore, but we should use 3+ characters for variable names. And ideally they should be self-descriptive

Reply:
I'm on the fence, so not a blocker. But yes I agree that a shorter scope makes it obvious what they are